### PR TITLE
[test] Add EfficientFormerL1

### PR DESCRIPTION
**Pull request by dayo09**
Let's add EfficientFormerL1 for our test model set.
Plus, use deep-copied model for export in test, because some model with BatchNorm have some issue to result in FakeTensor for inference.

---

From #93

### CODE REVIEW

File: test/modules/model/EfficientFormerL1/model.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import timm
+import torch
+
+
+class EfficientFormerL1(torch.nn.Module):
```
Comment (dayo09) [inline test/modules/model/EfficientFormerL1/model.py:5]: ### Test Passes ✔️ 

```sh
./ccex test -m EfficientFormer
RUN unit tests for model EfficientFormer ...
test.modules.model.EfficientFormerL1.model.EfficientFormerL1 (test.pt2_to_circle_test.test_model.test.modules.model.EfficientFormerL1.model) ... ok

----------------------------------------------------------------------
Ran 1 test in 18.537s

OK
```

**Review by seockho-kim — APPROVED** [review]
LGTM

File: test/pt2_to_circle_test/test_pt2_to_circle.py [hunk 2]
```diff
@@ -140,6 +140,10 @@ def verify_circle(circle_model_path: str, opt_circle_model_str: str):
 @print_name_on_exception
 def infer_nnmodule(model: torch.nn.Module, example_inputs: tuple):
     with torch.no_grad():
+        # Model should be frozen to compare the result with others.
+        # e.g. BatchNorm running_mean/running_var will be updated during training mode, thus changing the model behavior.
+        model.eval()
```
[the remark this answers is not in the document]
Comment (dayo09) [reply test/pt2_to_circle_test/test_pt2_to_circle.py:145]: @Seunghui98 That's a good question. 

It seems only BatchNorm family within our test cases are affected - only they fail the test.
WHY? - BatchNorm changes its internal values during inference if it's not set as eval() mode (training mode)
WHY It didn't failed before then? - Before, eval() was set outside this function (in convert_nnmodule_to_pt2), but this PR includes the change that `convert_nnmodule_to_pt2` is run on copied model.

[the remark this answers is not in the document]
Comment (Seunghui98) [reply test/pt2_to_circle_test/test_pt2_to_circle.py:145]: Thanks for your kind explanation! I understood it 🙂

**Comment by dayo09** [comment]
Let me seperate "test frame related" commits with subtle change :-D This PR will be revised to contain only the test.

**Comment by mhs4670go** [comment]
Let's remove "No merge" tag after landing #131.
